frontend/src/v5/ui/components/shared/userMenu/userMenu.component.tsx

```diff
@@ -19,11 +19,8 @@ import { FormattedMessage } from 'react-intl';
 import { useState } from 'react';
 
 import { AuthActionsDispatchers } from '@/v5/services/actionsDispatchers/authActions.dispatchers';
-import ContactUsIcon from '@assets/icons/email.svg';
-import InviteAFriendIcon from '@assets/icons/add_user.svg';
 import TeamspacesIcon from '@assets/icons/teamspaces.svg';
 import VisualSettingsIcon from '@assets/icons/settings.svg';
-import SupportCentreIcon from '@assets/icons/question_mark.svg';
```

Comment:
I guess this file is not needed anymore (same for add_user), so we may as well delete them

Reply:
Also, the link to the design indicates different icons for "teamspaces" as well as "settings", not sure this is meant to be addressed here. If not, ignore this comment